CONTRIBUTING.md

```diff
@@ -10,9 +10,9 @@ In 2014, you -- the Scala community -- matched the core team at EPFL in number o
 
```

Comment:
I can't commend directly on it, but line 9 references 2014. Do we have more recent data?

Reply:
I was going to say: It would be nice to say something as motivational, but the Scala 3 numbers would not support it. "Thank-you for your contributions since 2014. Would you mind forward-porting that to Scala 3, which was forked in 2014?"

Reply:
we don't have more recent data, sadly

Reply:
collecting this data is typically motivated by Adriaan preparing some conference slides :-)

Reply:
I remember those slides!